sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items-mlflow.ipynb

```diff
@@ -0,0 +1,1162 @@
+    "\n",
+    "\n",
+    "def read_image(image_path):\n",
+    "    with open(image_path, \"rb\") as f:\n",
+    "        return f.read()\n",
+    "\n",
+    "\n",
+    "request_json = {\n",
+    "    \"input_data\": {\n",
+    "        \"columns\": [\"image\"],\n",
+    "        \"data\": [base64.encodebytes(read_image(sample_image)).decode(\"utf-8\")],\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "request_file_name = \"sample_request_data.json\"\n",
+    "\n",
+    "with open(request_file_name, \"w\") as request_file:\n",
+    "    json.dump(request_json, request_file)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "resp = ml_client.online_endpoints.invoke(\n",
+    "    endpoint_name=online_endpoint_name,\n",
+    "    deployment_name=deployment.name,\n",
+    "    request_file=request_file_name,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualize detections\n",
+    "Now that we have scored a test image, we can visualize the bounding boxes for this image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.image as mpimg\n",
+    "from PIL import Image\n",
+    "import numpy as np\n",
+    "import json\n",
+    "\n",
+    "IMAGE_SIZE = (18, 12)\n",
+    "plt.figure(figsize=IMAGE_SIZE)\n",
+    "img_np = mpimg.imread(sample_image)\n",
+    "img = Image.fromarray(img_np.astype(\"uint8\"), \"RGB\")\n",
+    "x, y = img.size\n",
+    "\n",
+    "fig, ax = plt.subplots(1, figsize=(15, 15))\n",
+    "# Display the image\n",
+    "ax.imshow(img_np)\n",
+    "prediction = json.loads(resp)\n",
+    "score_threshold = 0.5\n",
+    "\n",
+    "label_offset_x = 30\n",
+    "label_offset_y = 30\n",
+    "for index, score in enumerate(prediction[\"probs\"]):\n",
+    "    if score > score_threshold:\n",
+    "        label = prediction[\"labels\"][index]\n",
+    "        display_text = \"{} ({})\".format(label, round(score, 3))\n",
+    "        print(display_text)\n",
+    "\n",
+    "        color = \"red\"\n",
+    "        plt.text(label_offset_x, label_offset_y, display_text, color=color, fontsize=30)\n",
+    "        label_offset_y += 30\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Delete the deployment and endopoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.online_endpoints.begin_delete(name=online_endpoint_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Next Steps\n",
```

Comment:
Please add the section similar to other notebooks here: "Next Step: Load the best model and try predictions"